src/Microsoft.OData.Core/Evaluation/ODataMetadataContext.cs

```diff
@@ -43,8 +43,9 @@ internal interface IODataMetadataContext
         /// </summary>
         /// <param name="resourceState">Resource state to use as reference for information needed by the builder.</param>
         /// <param name="useKeyAsSegment">true if keys should go in separate segments in auto-generated URIs, false if they should go in parentheses.</param>
+        /// <param name="isDelta">true if the payload being read is a delta payload</param>

```

Comment:
>payload [](start = 76, length = 7)

ends with '.'

Reply:
Fixed.

---
In reply to: [256641496](https://github.com/OData/odata.net/pull/1404#discussion_r256641496) [](ancestors = 256641496)